Assets/Scripts/NarrationItem.cs

```diff
@@ -1,23 +1,38 @@
 using System;
-
+using System.Collections.Generic;
+using System.ComponentModel;
 using UnityEngine;
-[Serializable]
-public class NarrationItem {
-    [SerializeField]
-    private string id;
-    [SerializeField]
-    private string story;
-    [SerializeField]
-    private string section;
-    [SerializeField]
-    private string triggered;
-    [SerializeField]
-    private string character;
-    [SerializeField]
-    private string line;
-    [SerializeField]
-    private string portrait;
-    [SerializeField]
-    private string context;
+using UnityEngine.UI;
+
+[CreateAssetMenu(fileName = "Narrative", menuName = "Narrative/Narrative Item")]
+public class NarrationItem : ScriptableObject{
+    [Tooltip("Day the narration occurs")]
+    public Day                 day;
+    [Tooltip("Character who is speaking")]
+    public Character character;
+    [Tooltip("Sounds that play in order from the beginning of the narration")]
+    public List<AudioClip> sounds;
+    [TextArea,Tooltip("Text that is spoken by the character")]
+    public string line;
+    [Tooltip("Next Narrative item. 1 or more")]
+    public List<NextNarrative> next;
+    [Tooltip("Image to be displayed behind the characters")]
+    public Image background;
 
 }
+
+[Serializable]
+public class NextNarrative {
+    [SerializeField, Tooltip("What triggers the next narration.")]
```

Comment:
So how does this integrated with the button already integrated?

Reply:
The next button would be directly linked to every narrative item that needs it for going to the next line.